tests/admin_views/tests.py

```diff
@@ -5841,7 +5841,9 @@ def test_prepopulated_fields(self):
         element = self.selenium.find_element(
             By.ID, "id_relatedprepopulated_set-2-0-status"
         )
-        self.selenium.execute_script("window.scrollTo(0, %s);" % element.location["y"])
+        self.selenium.execute_script(
+            "window.scrollTo(200, %s);" % element.location["y"]
+        )
```

Comment:
```suggestion
        self.selenium.execute_script(
            "window.scrollTo(%s, %s);"
            % (status_element.location["x"], status_element.location["y"])
        )
```

Reply:
Oh, I forgot the one I was testing with, thanks!